services/core-api/app/api/projects/major_mine_application/models/major_mine_application_document_xref.py

```diff
@@ -35,6 +35,8 @@ class MajorMineApplicationDocumentXref(Base):
     create_user = association_proxy('mine_document', 'create_user')
     versions = association_proxy('mine_document', 'versions')
     update_timestamp = association_proxy('mine_document', 'update_timestamp')
+    mine_document_bundle = association_proxy('mine_document', 'mine_document_bundle')
```

Comment:
Would want to avoid including the whole bundle with the document like in `project_summary_document_xref`.

Reply:
Oh no! I thought we took that out! Are we using it on the FE anywhere? If so, it's doing a much larger query than necessary, especially if it queries it every time.